srcpkgs/lainsafecli/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'lainsafecli'
+pkgname=lainsafecli
+version=0.4
+revision=1
+archs="noarch"
+create_wrksrc=yes
+depends="perl perl-LWP perl-LWP-Protocol-https"
+build_style=meta
+short_desc="Command line interface for lainsafe."
+maintainer="qorg11 <qorg@vxempire.xyz>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/qorg11/lainsafe"
+distfiles="https://github.com/qorg11/lainsafe/archive/${version}.tar.gz"
+checksum=6359da930c002178a8ae34278a1874697db25f9778b16038de732bcaab64c4ff
+
+do_install() {
+	cd lainsafe-0.4/ && make
```

Comment:
Don't do this either. The cd command is unnessecary if you remove the build_style and let it extract itself correctly. In addition, the Makefile has no default target so this does nothing.

The Makefile also does not respect `$DESTDIR` which `xbps-src` requires. IMO, reimplement the Makefile in xbps-src, with something like:

```
vbin lainsafecli
vman doc/lainsafecli.1
```

Reply:
The default Makefile target is install: https://github.com/qorg11/lainsafe/blob/master/Makefile

should I add a new targes (e.g. xbpsinstall) that does this:
~~~
vbin lainsafecli
vman doc/lainsafecli.1
~~~
?

Reply:
No, the makefile does not have a "default" target. Standard makefile behavior is for the default target to be `all`, it might just be a quirk of GNU make that it happens to run install. I also don't understand the `install: lainsafe` since there is no lainsafe target. Anyway, no, you should not add a new void-specific target. Those command are specific to xbps-src and will not work in your Makefile.

You have two options:

1. Bypass the makefile and use the vbin and vman commands directly in do_install.
2. Fix the Makefile so that it respects $DESTDIR and $PREFIX.

Reply:
The default target in any Makefile is the first listed, not `all` or some other special keyword.

Reply:
>Bypass the makefile and use the vbin and vman commands directly in do_install. 

Fixed by doing that.  
Now I only have to extract the tar file in the lainsafecli-${version} directory. By default it extracts it in the lainsafe-${version} directory. I guess it happens because the tar file has a lainsafe-${version} inside.